pkg/mock/nsxop.ini

```diff
@@ -6,3 +6,6 @@ nsx_api_managers = 127.0.0.1
 nsx_api_password = admin
```

Comment:
DEFAULT]
debug = True
[nsx_v3]
nsx_api_user = admin
nsx_api_password = admin
[ha]
#enable=true
[k8s]
[coe]
cluster = k8scl-one

It's good that we change nsxop.ini file as the formal ini format, so that others can also follow this way to add more UTS if necessary.

Reply:
OK